### PR TITLE
Add usage for fish shell

### PR DESCRIPTION
**Pull request by n10v**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,19 @@ Add to your `.zshrc`
...
 ```
 
+**fish**
+
+Add to your `config.fish`
+```bash
+function fish_prompt
+    set_color $fish_color_cwd
+    echo -n (prompt_pwd)
+    git-radar --bash -fetch
```
Comment (michaeldfallen) [inline README.md]: I know the `--bash` option will work for fish shellers but should we provide a `--fish`, initially as an alias of `--bash`. That way we can provide a native fish prompt in the future without changing the interface? 

All it needs is a change in [`git-radar`](https://github.com/michaeldfallen/git-radar/blob/master/git-radar#L85):

``` bash
if [[ "$command" == "--bash" || "$command" == "--fish" ]]; then
  $dot/prompt.bash
fi
```

**Comment by n10v** [comment]
Yes, I think it would be better

**Comment by michaeldfallen** [comment]
Closing as #23 has added this functionality

**Comment by n10v** [comment]
https://github.com/michaeldfallen/git-radar/pull/23 doesn't add this functionality 
This PR just edit README file, that https://github.com/michaeldfallen/git-radar/pull/23 don't

**Pull request closed without merging**